src/views/v2/events-bar/filters.php

```diff
@@ -13,3 +13,6 @@
  *
  */
 ?>
+<div class="tribe-events-c-events-bar__filters">
```

Comment:
I think, ideally it would be like you say but overriden by events filterbar. This addition has more to do with the re-structuring of the events bar and to have the different template partials "on the menu" to test things out.

Reply:
yes, i meant filter bar.

Reply:
Yeaps, we don't want any code here, it lives in Filterbar. File should stay empty.

Reply:
So, not even for demos? What about the mobile version of the navigation and the filters?

Reply:
Nopes, we need to make this customer ready. So if we need it for the demo, add it in Filterbar, which is the correct place.

Reply:
But filterbar is not ready for that. I thought we agreed last time that the styles would live here now.

Reply:
We still need to resolve this with @bordoni . @juanfra if needed, can you hop on a call with G and figure this out?

Reply:
I left it commented out, so we can move it once Filterbar is ready. Like what happened with ECP.